About

vivekpatel17 opened the pull request:
### **About** Section
- Added the about section.
- [x] JWOC Participant

Closes: #4 

### Describe the changes you've made
New feature **About** section has been added.
- Added **background** 
- some **dummy heading and  sub-headings**

### Type of change

- [x] New feature(non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)

### **Checklist:**

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x]  My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### **Screenshots**
![image](https://drive.google.com/uc?export=view&id=1SYVhY_bK00ob2bDLi7oeSB7fEHOpLFyX)
![image](https://drive.google.com/uc?export=view&id=1uRduL_Y0suoKX8Gwy81YUVa741CzPfSL)

Commit:
added about page

Commit:
changes in about

Commit:
added about

prathimacode-hub commented:
Kindly add a [PR TEMPLATE](https://github.com/prathimacode-hub/MedFlare/blob/main/.github/pullrequest_template.md) and fill up the details accordingly. Also the screenshot isn't visible too. @vivekpatel17

vivekpatel17 commented:
### **About** Section
- Added the about section.
- [x] JWOC Participant

Closes: #4 

### Describe the changes you've made
New feature **About** section has been added.
- Added **background** 
- some **dummy heading and  sub-headings**

### Type of change

- [x] New feature(non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)

### **Checklist:**

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x]  My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### **Screenshots**
![image](https://drive.google.com/uc?export=view&id=1SYVhY_bK00ob2bDLi7oeSB7fEHOpLFyX)
![image](https://drive.google.com/uc?export=view&id=1uRduL_Y0suoKX8Gwy81YUVa741CzPfSL)

prathimacode-hub reviewed (approved):
Congrats on your PR. It's approved. Welcome to repo. Fab work. This was a great start. Keep it up. 👏🎉@vivekpatel17
